### PR TITLE
Add MacOS/aarch64 builds to JDK11 pipelines

### PR DESCRIPTION
**Pull request by sxa**
As discussed in PMC meeting on March 30, 2022.

Draft at the moment as I've been getting hangs when enabling builds with ccache and need to verify it works ok without that. Approvals welcome for the change though :-)

Signed-off-by: Stewart X Addison <sxa@redhat.com>

### CODE REVIEW

File: pipelines/jobs/configurations/jdk11u_pipeline_config.groovy [hunk 1]
```diff
@@ -41,7 +41,6 @@ class Config11 {
                 arch                : 'x64',
                 dockerImage         : 'adoptopenjdk/alpine3_build_image',
                 test                : 'default',
```
Comment (karianna) [inline pipelines/jobs/configurations/jdk11u_pipeline_config.groovy:43]:
```suggestion
                test                : 'default'
```

**Review by karianna — APPROVED** [review]
LGTM

**Comment by sxa** [comment]
Wil require https://github.com/adoptium/temurin-build/pull/2885 to be merged before this can build successfully, so please do not merge until that is complete.

**Comment by sxa** [comment]
I've just merged the other PR so we can now try this

**Pull request merged**